pages/fundamentals/introduction.html

```diff
@@ -1,190 +1,181 @@
+Go는 많은 기능이 있습니다. 일부는 고유하게 작성된 것이고 일부는 다른 언어에서 차용하였습니다
 <ul>
 <li>
-	built-in concurrent programming support
+	내장 동시 프로그래밍 지원
 	<ul>
 	<li>
-		goroutines (green threads) and start new goroutines easily.
+		쉽게 사용할 수 있는 그린 스레드인 고루틴
 	</li>
 	<li>
-		channels (based on CSP model) and select mechanisms to do synchronizations between goroutines.
+    채널(CSP 모델 기반) 및 고루틴 간 동기화를 위한 <code>select</code> 메커니즘
 	</li>
 	</ul>
 </li>
 <li>
-	the container types <code>map</code> and <code>slice</code> are first-class citizens.
+	컨테이너 타입의 <code>map</code> 과 <code>slice</code> 는 일급 객체입니다.
 </li>
 <li>
-	polymorphism through interfaces.
+	인터페이스를 통한 다형성
 </li>
 <li>
-	value boxing and reflection through interfaces.
+	인터페이스를 통한 박싱 및 리플렉션
 </li>
 <li>
-	pointers.
+	포인터
 </li>
 <li>
-	function closures.
+	함수 클로저
 </li>
 <li>
-	methods.
+	메서드
 </li>
 <li>
-	deferred function calls.
+	디퍼드 함수 호출
 </li>
 <li>
-	type embedding.
+	타입 임베딩
 </li>
 <li>
-	type deduction.
+	타입 추론
 </li>
 <li>
-	memory safety.
+	메모리 안정성
 </li>
 <li>
-	automatic garbage collection.
+	자동 쓰레기 수집기(garbage collection)
 </li>
 <li>
-	great cross-platform compatibility.
+	크로스 플랫폼 호환성
 </li>
 <li>
-	custom generics (since Go 1.18).
+	사용자 정의 제네릭(Go 1.18 이상)
 </li>
 </ul>
 
-Besides the above features, further highlights are:
+위의 기능 외에도 추가 하이라이트는 다음과 같습니다
 <ul>
 <li>
-	The syntax of Go is deliberately designed to be simple, clean,
-	and similar to other popular programming languages.
-	This makes Go programming easy to pick up.
+	Go는 다른 프로그래밍 언어와 구문이 유사하고, 단순하고 깔끔하도록 설계되었습니다.
+	이를 통해 Go 프로그래밍을 쉽게 익힐 수 있습니다.
 </li>
 <li>
-	Go comes with a great set of standard code packages which provide all kinds of
-	common functionalities. Most of the packages are cross-platform.
+	Go는 모든 종류의 공통 기능을 제공하는 훌륭한 표준 코드 패키지 세트와 함께 제공됩니다.
+	대부분의 패키지는 크로스 플랫폼입니다.
 </li>
 <li>
-	Go also has an active community, and there are
-	<a href="https://github.com/avelino/awesome-go">plenty of
-	high quality third party Go packages and projects</a> to import and use.
+	Go에는 활발한 커뮤니티도 있고
+	<a href="https://github.com/avelino/awesome-go">많은
+	고품질의 타사 Go 패키지 및 프로젝트</a>를 가져와서 사용할 수 있습니다.
 </li>
 </ul>
 
 </div>
 
 <p>
-Go programmers are often called gophers.
+Go 프로그래머는 고퍼들(Gophers)이라고 불립니다.
 </p>
 
 <p>
-In fact, although Go is a compiled and static typed programming language,
-Go also has many features which are usually only available in dynamic script languages.
-It is hard to combine these two kinds into one language, but Go did it.
-In other words, Go owns both the strictness of static languages
-and the flexibility of dynamic languages.
-I can't say there are not any compromises between the two,
-but the effect of the compromises is much weaker than the benefits of the combination in Go.
+실제로 Go는 컴파일되고 정적 타입의 프로그래밍 언어이지만 Go에서는 일반적으로 동적 스크립트 언어에서만 사용할 수 있는 많은 기능도 있습니다.
+두 종류를 하나의 언어로 합치는 것은 어렵지만 Go에서는 해냈습니다.
+즉, Go는 정적 언어의 엄격함과 동적 언어의 유연성을 모두 가지고 있습니다. 둘 사이에 문제가 없었다고 말할 수는 없지만 Go에서의 이점 보다 훨씬 덜합니다.

```

Comment:
뒤 문장이 뭔가 좀 어색한 것 같습니다. 조금 더 직역을 하는 게 덜 어색할 듯 합니다.

```suggestion
즉, Go는 정적 언어의 엄격함과 동적 언어의 유연성을 모두 가지고 있습니다.
두 개념 사이의 절충이 없었다고는 할 수 없지만, 그 영향은 이 둘을 합치며 얻은 이득보다는 훨씬 미미합니다.
```